website/public/css/tasks.styl

```diff
@@ -394,6 +394,11 @@ for $stage in $stages
   a.badge
     position: relative; top:-2px
 
+  a.push-down span
+    -ms-transform: rotate(180deg); /* IE 9 */
+    -webkit-transform: rotate(180deg); /* Chrome, Safari, Opera */
+    transform: rotate(180deg);
+
```

Comment:
Gotcha. Is it okay for me to make an upside down version of the icon? 

I have added the `push-down` class when ctrl/cmd is clicked. The link doesn't have a class in the push up stated.